keycloak/keycloak-admin-ui
# Test email connection

## Motivation

https://issues.redhat.com/browse/APPDUX-936

## Brief Description


## Verification Steps


1. From your local version of https://github.com/keycloak/keycloak, `cd` to `testsuite/utils` 
2. Run `mvn exec:java -Pmail-server` 
3. Go to Realm settings -> email
4. Enter `localhost` in the "Host" form field.
5. Enter `3025` in the "Port" form field.
6. Click the "Test Connection" button on the bottom of the page.
7. Verify the network call is successful and the UI displays a success alert.

## Checklist:


## Additional Notes
